Rakefile

```diff
@@ -223,3 +223,40 @@ namespace :crowbar do
     end
   end
 end
+
+namespace :test do
+  task dependencies: [] do
+    if ENV["TRAVIS"]
+      system("pip install bashate")
+    else
+      system("sudo pip install bashate")
+    end
+    exit $?.exitstatus
+  end
+
+  desc "Run bashate tests"
+  task bashate: [:dependencies] do
+    script = "scripts/install-chef-suse.sh"
```

Comment:
As this is anyway already a variable name it can be extended more than easy. But a loop for a single file is some kind of weird...